wradlib/georef/xarray.py

```diff
@@ -210,6 +210,10 @@ def georeference_dataset(obj, **kwargs):
         dimlist = list(obj["elevation"].dims)
     else:
         dimlist = list(obj["azimuth"].dims)
+
+    if xyz.ndim > 3:
```

Comment:
Are other Datasets/DataArrays possible which have more than 3 dimensions where this doesn't fit? We might need to check the dimensions here.

Reply:
xyz is an array of cartesian coordinates for every spherical coordinate, so the possible dimensions are: elevation, azimuth, range, 3. For 2d, it either has (elevation, range, 3) or (azimuth, range, 3) dimensions. For 3d, the only option is the full (elevation, azimuth, range, 3) dimensions. Thus, i think adding these lines for the 3d case will not break other functionalities

Reply:
> xyz is an array of cartesian coordinates for every spherical coordinate, so the possible dimensions are: elevation, azimuth, range, 3. For 2d, it either has (elevation, range, 3) or (azimuth, range, 3) dimensions. For 3d, the only option is the full (elevation, azimuth, range, 3) dimensions. 

Ah, yes, great! Please add your above quoted explanation above the if-clause as comment for further reference. Thanks!